.github/workflows/push.yml

```
@@ -1,4 +1,8 @@
-on: push
+on:
+  push:
+    tags:
+      - '*'
```

A trivial change for consistency:

```suggestion
    - '*'
```